README.md

```diff
@@ -27,6 +27,19 @@ CREATE TABLE paypal (
   currency TEXT,
   country_code TEXT
 );
+
+CREATE TABLE stripe (
+  id TEXT PRIMARY KEY,
+  timestamp TIMESTAMP WITH TIME ZONE,
+  amount INTEGER,
```

Comment:
if the transaction is in USD, stripe reports the value in cents, so 10.10 would be 1010 cents